towhee/engine/operator_io/_operator_io.py

```diff
@@ -14,7 +14,7 @@
 
```

Comment:
Do we still need this file, everything has moved to reader, init, and writer

Reply:
We needn't this file, I forgot to delete it.